src/components/Controls/Controls.jsx

```diff
@@ -0,0 +1,33 @@
+import { filterDropDown } from "../../utils/utils";
+
+function Controls({ abilities, selectedAbility, handleChange }){
+    const filter = filterDropDown(abilities, selectedAbility)
+    console.log('DROP', selectedAbility)
+    return (
+        <>
+            <div>
+                <h1>Filter By Ability:</h1>
+                <select value={selectedAbility} onChange={(e) => handleChange(e.target.value)}>
+                    <option value='all'>All</option>
+                    {filter.map(category => (
+                        <option key={category} value={category}>{category}</option>
+                        ))}
+                </select>
+            </div>
+            <div>
+                <h1>Ability:</h1>
+                <h1>{`${selectedAbility}`}</h1>
+            </div>
+            <div>
+                <h1>Sort By Pokemon:</h1>
+                <label>Ascending
+                    <input type="radio" name="sort" value="ascending" />
```

Comment:
Since this functionality isn't working, I'd comment out this code so it isn't showing on the page for users